File: appRvSample/src/main/java/in/til/yogeshkumar/timesanimation/adapter/FullScreenAdapter.java
package in.til.yogeshkumar.timesanimation.adapter;

import android.app.Activity;
import androidx.annotation.NonNull;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import in.til.yogeshkumar.timesanimation.R;

/**
 * Created by Yogesh Kumar.
 */
public class FullScreenAdapter extends RecyclerViewAdapter {

    public FullScreenAdapter(Activity context) {
        super(context);
    }

    @NonNull
    @Override
    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_card_fill, parent, false);
        return new ViewHolder(itemView);
    }

    @Override
    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
        super.onBindViewHolder(holder, position);
    }
}

File: appRvSample/src/main/java/com/timesanimation/adapter/FullScreenAdapter.kt
package com.timesanimation.adapter

import com.timesanimation.R
import android.app.Activity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

/**
 * Created by Yogesh Kumar.
 */
class FullScreenAdapter(context: Activity) : RecyclerViewAdapter(context) {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
        val itemView: View = LayoutInflater.from(parent.context).inflate(R.layout.item_card_fill, parent, false)
        return ViewHolder(itemView)
    }

    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
        super.onBindViewHolder(holder, position)
    }
}